src/portable/nuvoton/nuc121/dcd_nuc121.c

```diff
@@ -153,7 +158,7 @@ static void dcd_in_xfer(struct xfer_ctl_t *xfer, USBD_EP_T *ep)
   else
 #endif
   {
-    memcpy((uint8_t *)(USBD_BUF_BASE + ep->BUFSEG), xfer->data_ptr, bytes_now);
+    usb_memcpy((uint8_t *)(USBD_BUF_BASE + ep->BUFSEG), xfer->data_ptr, bytes_now);
```

Comment:
to be honest, I feel a bit odd about this change, would it work if we cast the buffer with `volatile uint8_t*` instead of uint8_t*

Reply:
`volatile` doesn't work as ` Error[Pe167]: argument of type "uint8_t volatile *" is incompatible with parameter of type "void *"` 

In fact read setup packet from buffer with `memcpy` (LDM instruction) works, only write doesn't work.

I read the manual, it didn't mention the access type.

Reply:
I checked the manual as well, it is not mentioned. But It is very likely that USB SRAM is byte access. So yeah, this PR is spot on and needed. 
https://github.com/majbthrd/nuc_driver/blob/master/nuc126/StdDriver/inc/usbd.h#L524

Only 1 request, would you mind adding an comment at this line and usb_memcpy below to explain the reason something like `although it is not mentioned in manual, but the USB SRAM seems to only support byte access and memcpy could possibly do it by words etc ...`